.devcontainer/post_create.sh

```diff
@@ -0,0 +1,26 @@
+# For writing commands that will be executed after the container is created
+
+# Installs `human_hip` as local library without resolving dependencies (--no-deps)
+python3 -m pip install -e ".[all]"
```

Comment:
The current research image already have `braingeneerspy` installed. So it's probably best to add a step above this to uninstall the current installed version first.

Reply:
Done

Reply:
Done